js/app/src/Blocks.svelte

```diff
@@ -45,6 +45,9 @@
 
 	let loading_status = create_loading_status_store();
 
+	const rendered_node_ids = new Set();
```

Comment:
Maybe we can change this to `walked_node_ids` or something. `render` and `mount` mean the same thing to me.